CesiumGltfReader/test/data/DucksMeshopt/README.md

```diff
@@ -0,0 +1,7 @@
+# GLTF Model for Testing
+
+This repository contains the Duck GLTF model that is used for testing various features and scenarios. The model is taken from the [glTF-Sample-Assets](https://github.com/KhronosGroup/glTF-Sample-Assets) repository, which provides a set of sample models for testing and demonstrating the glTF format.
+
+## License Information
+
+The Duck model is in the public domain and can be used for any purpose. The source is [here](https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/Duck).
```

Comment:
The license on the linked page definitely doesn't say it's in the public domain! 😬